### ISSUE TITLE
Accessibility issue: Some links are missing a name

### ISSUE THREAD

**Issue by chichiwang** [chichiwang/exercise-js#155]
The logo, and the links on the section headings within the boilerplate UI are missing names (since the links only wrap an icon/logo). Users of screen readers need the links to be more descriptive so they can properly navigate the site. Use [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) and/or [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to properly label these links.

Acceptance criteria:
* Logo link passes accessibility requirements
* Heading links pass accessibility requirements

**Comment by chichiwang** [comment]
Task completed with PR #158
